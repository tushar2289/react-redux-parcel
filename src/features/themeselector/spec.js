import themeReducer, { setTheme } from './slice';

describe('Theme selector', () => {
  let initialState;
  beforeEach(() => {
    initialState = themeReducer(undefined, {});
  });

  it('should return initial state', () => {
    expect(initialState).toMatchSnapshot();
  });

  it('should set theme', () => {
    expect(
      themeReducer(initialState, {
        type: setTheme.type,
        payload: 'dark',
      })
    ).toMatchSnapshot();
  });
});
