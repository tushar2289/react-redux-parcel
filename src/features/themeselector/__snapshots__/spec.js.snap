// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Theme selector should return initial state 1`] = `
Object {
  "theme": null,
}
`;

exports[`Theme selector should set theme 1`] = `
Object {
  "theme": "dark",
}
`;
